### DISCUSSION TITLE
Do I need to update the server?

### DISCUSSION THREAD

**Discussion by LuukLHD** [LsHallo/ets2-dedicated-convoy-server#3] [General]
I have made a server and it's not online after an update of ets2 itself.

**Comment by LsHallo** [comment]
Yes it needs to be updated. Currently you can't update yourself and rely on me. I will merge my PR soon and then the server will auto update if you set the environment variable correctly.

**Reply by LsHallo** [reply]
Please use v0.2.2 (or latest) of the image and set `ETS_SERVER_UPDATE_ON_START` to true. Then it should work as expected.

**Reply by LuukLHD** [reply]
Thanks, I appreciate it

**Reply by LuukLHD** [reply]
I can't get it online right now. If I start my docker container (even after restarting my server and reinstalling the docker image) it doesn't show up in the game. Do you know what I'm doing wrong?

**Reply by LsHallo** [reply]
Are you on a rented server with as dedicated ipv4?
If you are behind a NAT you need to forward ports 27015-27016 TCP+UDP. 
Docker network mode host is also important to be visible in the server list.
Also check your firewall settings to allow 27015-27016 TCP+UDP.
Does the server start up fine? Can you post the last 20-30 lines of your server log? (Maybe censor search id).
Have you tried searching for the direct search id in ETS and does it work that way?

**Reply by LuukLHD** [reply]
Nevermind, it's working now. I just forgot to change one simple thing. Anyways, it's amazing. Thank you for making this possible.
